packages/cli/src/ceramic-daemon.ts

```diff
@@ -307,154 +189,105 @@ class CeramicDaemon {
    */
-  async multiQuery (req: Request, res: Response, next: NextFunction): Promise<void> {
+  async multiQuery (req: Request, res: Response): Promise<void> {
     const { queries } = <MultiQueries> req.body
-    try {
-      const results = await this.ceramic.multiQuery(queries)
-      const response = Object.entries(results).reduce((acc, e) => {
-        const [k, v] = e
-        acc[k] = DoctypeUtils.serializeState(v.state)
-        return acc
-      }, {})
-      res.json(response)
-    } catch (e) {
-      return next(e)
-    }
-    next()
+    const results = await this.ceramic.multiQuery(queries)
+    const response = Object.entries(results).reduce((acc, e) => {
+      const [k, v] = e
+      acc[k] = DoctypeUtils.serializeState(v.state)
+      return acc
+    }, {})
+    res.json(response)
   }
 
   /**
    * Pin document
    */
-  async pinDocument (req: Request, res: Response, next: NextFunction): Promise<void> {
+  async pinDocument (req: Request, res: Response): Promise<void> {
     const docId = DocID.fromString(req.params.docid)
-    try {
-      await this.ceramic.pin.add(docId)
-      res.json({ docId: docId.toString(), isPinned: true })
-    } catch (e) {
-      return next(e)
-    }
-    next()
+    await this.ceramic.pin.add(docId)
+    res.json({ docId: docId.toString(), isPinned: true })
   }
 
   /**
    * Unpin document
    */
-  async unpinDocument (req: Request, res: Response, next: NextFunction): Promise<void> {
+  async unpinDocument (req: Request, res: Response): Promise<void> {
     const docId = DocID.fromString(req.params.docid)
-    try {
-      await this.ceramic.pin.rm(docId)
-      res.json({ docId: docId.toString(), isPinned: false })
-    } catch (e) {
-      return next(e)
-    }
-    next()
+    await this.ceramic.pin.rm(docId)
+    res.json({ docId: docId.toString(), isPinned: false })
   }
 
   /**
    * List pinned documents
    */
-  async listPinned (req: Request, res: Response, next: NextFunction): Promise<void> {
+  async listPinned (req: Request, res: Response): Promise<void> {
     let docId: DocID;
     if (req.params.docid) {
       docId = DocID.fromString(req.params.docid)
     }
-    try {
-      const pinnedDocIds = []
-      const iterator = await this.ceramic.pin.ls(docId)
-      for await (const id of iterator) {
-        pinnedDocIds.push(id)
-      }
-      res.json({ pinnedDocIds: pinnedDocIds })
-    } catch (e) {
-      return next(e)
+    const pinnedDocIds = []
+    const iterator = await this.ceramic.pin.ls(docId)
+    for await (const id of iterator) {
+      pinnedDocIds.push(id)
     }
-    next()
+    res.json({ pinnedDocIds: pinnedDocIds })
   }
 
-  async _notSupported (req: Request, res: Response, next: NextFunction): Promise<void> {
+  async _notSupported (req: Request, res: Response): Promise<void> {
     res.status(400).json({ status: 'error', message: 'Method not supported by read only Ceramic Gateway' })
```

Comment:
I think this json should match the format that we have in the `errorHandler` middleware.

Reply:
Yes.